Cargo.toml

```diff
@@ -25,7 +25,7 @@ chrono = "0.4"
 clap = "2"
 download = { path = "download" }
 error-chain = "0.12"
-effective-limits = "0.4"
+effective-limits = { git = "https://github.com/tblah/effective-limits.rs.git", rev = "4873a008d2b441b2d47ce01f62c6f5ec9c157df9" }
```

Comment:
if bump libc, why not `cargo update -p libc` and remove this change.

Reply:
The libc changes aren't in a released version yet so I think I have to tell cargo to fetch libc master branch instead of a release.